modules/Bio/EnsEMBL/Analysis/Hive/Config/FinaliseCoreDB.pm

```diff
@@ -0,0 +1,570 @@
+    'release_number'     => '' || $self->o('ensembl_release'),
+    'uniprot_set'        => '',                                                                                                # e.g. mammals_basic, check UniProtCladeDownloadStatic.pm module in hive config dir for suitable set,
+    'output_path'        => '',                                                                                                # Lustre output dir. This will be the primary dir to house the assembly info and various things from analyses
+    'assembly_name'      => '',                                                                                                # Name (as it appears in the assembly report file)
+    'assembly_accession' => '',                                                                                                # Versioned GCA assembly accession, e.g. GCA_001857705.1
+    'stable_id_prefix'   => '',                                                                                                # e.g. ENSPTR. When running a new annotation look up prefix in the assembly registry db
+    'stable_id_start'    => '0',                                                                                               # When mapping is not required this is usually set to 0
+    'skip_projection'    => '0',                                                                                               # Will skip projection process if 1
+    'skip_rnaseq'        => '0',                                                                                               # Will skip rnaseq analyses if 1
+    'mapping_required'   => '0',                                                                                               # If set to 1 this will run stable_id mapping sometime in the future. At the moment it does nothing
+    'mapping_db'         => '',                                                                                                # Tied to mapping_required being set to 1, we should have a mapping db defined in this case, leave undef for now
+    'uniprot_version'    => 'uniprot_2019_04',                                                                                 # What UniProt data dir to use for various analyses
+    'protein_entry_loc'  => catfile( $self->o('base_blast_db_path'), 'uniprot', $self->o('uniprot_version'), 'entry_loc' ),    # Used by genscan blasts and optimise daf/paf. Don't change unless you know what you're doing
+
+########################
+# Pipe and ref db info
+########################
+
+    'projection_source_db_name'         => '',                                                                                 # This is generally a pre-existing db, like the current human/mouse core for example
+    'projection_source_db_server'       => 'mysql-ens-mirror-1',
+    'projection_source_db_port'         => '4240',
+    'projection_source_production_name' => '',
```

Comment:
This is not used in the pipeline and can be removed

Reply:
Ah yes, only needed the db name.
Done